README.md

```diff
@@ -255,6 +257,36 @@ var awsProvisioner = new taskcluster.AwsProvisioner(options);
  * `awsProvisioner.backendStatus() : result`
  * `awsProvisioner.ping() : void`
 
+### Methods in `taskcluster.EC2Manager`
+```js
+// Create EC2Manager client instance with default baseUrl:
+//  - localhost:5555/v1
+var eC2Manager = new taskcluster.EC2Manager(options);
+```
+ * `eC2Manager.listWorkerTypes() : result`
+ * `eC2Manager.runInstance(workerType, payload) : void`
+ * `eC2Manager.terminateWorkerType(workerType) : void`
+ * `eC2Manager.workerTypeStats(workerType) : result`
+ * `eC2Manager.workerTypeHealth(workerType) : result`
+ * `eC2Manager.workerTypeErrors(workerType) : result`
+ * `eC2Manager.workerTypeState(workerType) : result`
```

Comment:
All these `workerType` APIs are declared experimental, right?

If ec2-manager is supposed to be a frontend for EC2, then it shouldn't be `workerType` aware. It might deal with pools of machines or some other concept, maybe just machines with tags... So we should clean up these APIs at some point, right?

Reply:
note: this is not something you have to fix before publishing a highly experimental API.